File: dart/lib/BluefinTecsMerchantServicesError.dart
class BluefinTecsMerchantServicesError extends Error {
  final bool isBluefinTecsMerchantServicesError = true;

  final String sdk = 'BluefinTecsMerchantServices';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinTecsMerchantServicesError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinTecsMerchantServicesError: ' + code + ': ' + message;
}

File: js/src/BluefinTecsMerchantServicesError.js


class BluefinTecsMerchantServicesError extends Error {

  isBluefinTecsMerchantServicesError = true

  sdk = 'BluefinTecsMerchantServices'

  constructor(code, msg, ctx) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

module.exports = {
  BluefinTecsMerchantServicesError
}

